adoc/reference-webui-systems.adoc

```diff
@@ -2669,19 +2671,20 @@ To create more activation keys, repeat the steps above.
 
 After creating the unique key, it appears in the list of activation keys along with the number of times it has been used.
 Only Activation Key Administrators can see this list.
-At this point, you can configure the key further, for example, associate the key with child channels (for example, the Tools child channel), packages (for example, the [package]#rhncfg-actions#  package) and groups.
+At this point, you can configure the key further.
+For example, associate the key with packages (for example, the [package]#rhncfg-actions#  package) and groups.
 Systems registered with the key get automatically subscribed to them.
 
-To change the settings of a key, click the key's description in the list to display its [guimenu]``Details`` page. Via additional tabs you can select child channels, packages, configuration channels, group membership and view activated systems.
+To change the settings of a key, click the key's description in the list to display its [guimenu]``Details`` page. Via additional tabs you can select packages, configuration channels, group membership, and view activated systems.
```

Comment:
I have a personal vendetta against 'via'! Perhaps: "Use additional tabs to select packages, ..." ?

Reply:
@Loquacity: from a mother language like you, now I am interested in the reason of being against the `via`. Maybe we are taught and we use it not that properly?

Reply:
> I have a personal vendetta against 'via'! Perhaps: "Use additional tabs to select packages, ..." ?

Ok, there are many more such vias in this file...

Reply:
@Loquacity pretty clear explanation, thanks a lot.
And you know, I'm Italian, with Latin flowing in veins, I am pretty sure I will break this rule at any time soon :sweat_smile:

Reply:
And I'm Australian, we are notorious for being mono-lingual! Not havin' any o' those forren words, round 'ere, mate!